examples/0_Getting_started.py

```diff
@@ -93,7 +93,11 @@
 # consolidate the cells above into a single step:
 
 # + tags=[]
-query = queries.MaterialImpactedSubstancesQuery().with_material_ids(["plastic-abs-high-impact"]).with_legislations(["EU REACH - The Candidate List"])  # noqa: E501
```

Comment:
I deliberately haven't run black on this cell in the past because the syntax is a bit weird for non-experienced Python users, at least until it's introduced in the cell following. Could you revert and add `# fmt: off` and `# fmt: on` around this cell?